api/host.go

```diff
@@ -102,15 +101,11 @@ func ModifyHostStatus(ctx context.Context, env evergreen.Environment, queue ambo
 	}
 
 	if newStatus == evergreen.HostTerminated {
-		if !queue.Info().Started {
```

Comment:
@Kimchelly @sophstad does this error seem needed? Looks like it was introduced a long time ago, but we don't seem to do this checking anywhere else.

Reply:
I don't think this check matters, there is a tiny case where the queues can be stopped as the app server shuts down (so `Started` is false), preventing more jobs from queueing. That being said, the enqueue will fail and the user will just get an error back anyways if the queue is down due to the app server shutdown, so it's just duplicate error checking.